### ISSUE TITLE
npo.nl links not working

### ISSUE THREAD

**Issue by stan-is-laus** [streamlink/streamlink#1160]
*Thanks for reporting an issue!*
*Please read the contribution guidelines first! (see the link above)*
*Also check the list of known issues before reporting an issue!*

*Feel free to use the following template. Be as detailed as possible.*
*Please see the text preview to avoid unnecessary formatting errors.*
*Don't forget to remove this text before submitting.*

----

### Checklist

- [x] This is a bug report.
- [ ] This is a feature request.
- [ ] This is a plugin (improvement) request.
- [ ] I have read the contribution guidelines.

### Description

When I want to watch anything from npo.nl it gives the same error message

### Expected / Actual behavior

Expect streamlink to open VOD in VLC but gives error in Terminal

### Reproduction steps / Explicit stream URLs to test

This VOD link doesn't work, although I can open it in my webbrowser and watch the video:
```
# streamlink https://www.npo.nl/nos-journaal/07-07-2017/POW_03375651:
# [cli][info] Found matching plugin npo for URL https://www.npo.nl/nos-journaal/07-07-2017/POW_03375651
# error: No playable streams found on this URL: https://www.npo.nl/nos-journaal/07-07-2017/POW_03375651
```

This VOD link, from zapp.nl, does work with the same plugin (npo):
```
# streamlink https://zappelin.nl/10-voor/gemist/VPWON_1271522:
# [cli][info] Found matching plugin npo for URL https://www.zapp.nl/topdoks/gemist/VPWON_1276930
# [cli][info] Available streams: 270p_alt (worst), 270p, 360p_alt, 360p, 432p_alt, 432p, 576p_alt, 576p (best)
```

Live stream also doesn't work:
```
# stasiuz$ streamlink https://www.npo.nl/live best
# [cli][info] Found matching plugin npo for URL https://www.npo.nl/live
# error: No playable streams found on this URL: https://www.npo.nl/live

```

### Environment details

Operating system and version: macOS Sierra 10.12.5
Streamlink and Python version: 0.7.0 // 2.7.10
VLC version: 2.2.6

### Comments, logs, screenshots, etc.

None of the NPO.NL links work.

[Love Streamlink? Please consider supporting our collective. Thanks!](https://opencollective.com/streamlink/donate)

**Comment by acz13** [comment]
I think they put some DRM or something on npo.nl?

**Comment by stan-is-laus** [comment]
Yes, I'm in the Netherlands. My friend has a similar setup same versions and OS, also in The Netherlands and his is working normally.

**Comment by acz13** [comment]
Weird, they're working for me too (now that I got around to updating streamlink...)

**Comment by stan-is-laus** [comment]
Sorry, I'm not yet familiar with the debugging syntax. Should I just enter:
__
streamlink -l debug
__
?

**Comment by acz13** [comment]
something like

`streamlink https://www.npo.nl/nos-journaal/07-07-2017/POW_03375651 -l debug`

**Comment by stan-is-laus** [comment]
Still the same:

```
# streamlink https://www.npo.nl/nos-journaal/07-07-2017/POW_03375651 -l debug
# [cli][info] Found matching plugin npo for URL https://www.npo.nl/nos-journaal/07-07-2017/POW_03375651
# error: No playable streams found on this URL: https://www.npo.nl/nos-journaal/07-07-2017/POW_03375651
```

**Comment by back-to** [comment]
the issue here is, you are using the **stable version 0.7.0**

There is an update on the **master branch** #1084

---

- You can wait for streamlink 1.0.0 #1115 

or

- change your file locally

or
- maybe the best way if you used **pip** to install streamlink

```
pip install -U git+https://github.com/streamlink/streamlink.git
```
*maybe you need to uninstall it before you can install it like this*

---

```
streamlink https://www.npo.nl/nos-journaal/07-07-2017/POW_03375651 best
[cli][info] Found matching plugin npo for URL https://www.npo.nl/nos-journaal/07-07-2017/POW_03375651
[cli][info] Available streams: vod, vod_alt2, vod_alt, 270p_alt (worst), 270p, 360p_alt, 360p, 432p_alt, 432p, 576p_alt, 576p (best)
[cli][info] Opening stream: 576p (hls)
[cli][info] Starting player: /usr/bin/smplayer
```

**Comment by back-to** [comment]
can be closed,
it should work with the **stable** version now #1215
